continuous-integration.md

```diff
@@ -0,0 +1,31 @@
+Continuous Integration
+====
+
+Continuous Integration (CI) is a practice where developers frequently push their code into a shared repository. 
+Automated builds and checks run on each check-in, allowing teams to detect problems early.
+
+## Tasks
+
+You should have continuous integration set up in your repository running the following tasks where relevant:
+* Running unit and integration tests
+* Building your application
+* Linting your codebase
+* Publishing artifacts
+* Other appropriate tasks for your repository
+
+## Platforms
+
+* Use TeamCity to run continuous integration tasks
+* If possible, have a centralised script in the repository named `scripts/ci`  
```

Comment:
minor: Could we add the qualifier that CI should have a single step, and that is to run this script:

`Where possible, have CI execute a single, centralised script in the repository named scripts/ci`

Reply:
Yeah, good call. Added this. Also corrected my own error of `scripts` -> `script` in the process